site/installing.md

```diff
@@ -63,6 +64,37 @@ This script downloads and runs a recent Scope docker image from the Docker Hub.
 
 After Scope has been launched, open your web browser to [https://cloud.weave.works](https://cloud.weave.works) and login. Click 'View Instance' in the top right-hand corner to see the Scope user interface.
 
+
+###<a name="cluster-no-net"></a>Installing Scope on a Local Cluster Without Weave Net
+
+This example assumes that you have a local cluster that is not using Weave Net, and which also has no special hostnames or DNS settings. Only the IP addresses assigned to it will be used to configure Scope. You will launch Scope using the IP addresses of all the nodes in the cluster. 
+
+Suppose you have the following cluster:
+
+ 192.168.100.16
+ 192.168.100.17
+ 192.168.100.18
+ 192.168.100.19
+ 192.168.100.20
+ 
+ In the steps that follow, you will manually peer each node with all of the other nodes during Scope launch. 
+ 
+**1. To begin run the following on each node:**
+
+     sudo curl -L git.io/scope -o /usr/local/bin/scope
+     sudo chmod a+x /usr/local/bin/scope
+
+ **2. Then on the first node:**
+
+     scope launch 192.168.100.18 192.168.100.19 192.168.100.20
+
+ **3. And on the second node:**
+
+     scope launch 192.168.100.17 192.168.100.20 192.168.100.21
+     scope launch 192.168.100.17 192.168.100.18 192.168.100.21
```

Comment:
There are 5 nodes in this example, shouldn't this say "And on the other nodes" and then give examples of how to construct the `scope launch` commands appropriately?